test/quicer_SUITE.erl

```diff
@@ -188,7 +188,6 @@ tc_stream_client_init(Config) ->
     listener_ready ->
       {ok, Conn} = quicer:connect("localhost", Port, [], 5000),
       {ok, Stm} = quicer:start_stream(Conn, []),
-      {ok, {_, _}} = quicer:sockname(Stm),
```

Comment:
why removed?

Reply:
Previouosly quicer:sockname(Stream) returned sockname of Connection. But we can assume we always have Connection which has created Stream?

Reply:
Yes. The stream should always have a connection that runs on.

Reply:
I thought we should only allow sockname(Connection) or sockname(Listener), because sockname(Stream) was actually sockname(Connection).

Reply:
I agree in terms of calling msquic API.

But in reality,  The application build on top of quicer might assume the same behavior as when they deal with socket, 
like the following:

```
inet:sockname(socket())
```

So they expect to call:

```
quicer:sockname(stream_handler())
```

I think it is ok to return the sock name of connection even the argument is a stream. 

In the stream ctx, there is a pointer to the connection ctx that can be referred to.

Reply:
Now I got it. fixed